spec/shopify_transporter/pipeline/magento/order/discounts_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require 'shopify_transporter/pipeline/magento/order/top_level_attributes'
+
+module ShopifyTransporter::Pipeline::Magento::Order
+  RSpec.describe Discounts, type: :helper do
+    context '#convert' do
+      it "extract fixed amount discount into correct discount codes in Shopify" do
+        magento_order = FactoryBot.build(:magento_order, :with_fixed_amount_discount)
+        shopify_order = described_class.new.convert(magento_order, {})
+        expected_discount_code = [
+          {
+            amount: 15,
+            code: 'TEST_DISCOUNT_CODE',
```

Comment:
Personally I'd replace `'TEST_DISCOUNT_CODE'` with `magento_order['discount_description']` or whatever, because that way if you change the factory you dont need to also change the tests to match.

There's definitely an argument for keeping simpler tests though, and this is the nittiest of picks. Do whatever you think is better, this is non-blocking.